packages/sveltekit/src/builder.ts

```diff
@@ -19,14 +19,17 @@ async function convertSvelteKitRequest(request) {
 
 export class SvelteKitBuilder extends BaseBuilder {
   constructor(config?: Partial<SvelteKitConfig>) {
+    const workingDir = config?.workingDir || process.cwd();
+    const dirs = getWorkflowDirs({ dirs: config?.dirs });
+
     super({
       ...config,
-      dirs: ['workflows'],
+      dirs,
```

Comment:
this still limits people to having their workflows inside `workflows` dir. do we want something like this to scan the whole project?

```suggestion
      dirs: [.]
```

Reply:
idk what this suggestion means

see what nextjs does. It's enough to just scan whatever directory the server routes will be in, since everything gets imported from there